editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/index.tsx

```diff
@@ -139,7 +139,7 @@ const DesignSettings: React.FC = () => {
           <ThemeAndLogoForm team={team} onSuccess={() => setOpen(true)} />
           <ButtonForm team={team} onSuccess={() => setOpen(true)} />
           <TextLinkForm team={team} onSuccess={() => setOpen(true)} />
-          <FaviconForm />
+          <FaviconForm team={team} onSuccess={() => setOpen(true)} />
```

Comment:
There's a little repetition with some of these props and form behaviours - I'll look at a small follow on PR to try and tidy this up 👌